src/components/BookForm.js

```diff
@@ -3,7 +3,7 @@ import { v4 as uuidv4 } from 'uuid';
 import { useDispatch } from 'react-redux';
 import { addBookToAPI } from '../redux/books/books';
 
-const categories = ['Action', 'Science Fiction', 'Economy'];
+const categories = ['Action', 'Fiction', 'Romance', 'Politics', 'Science'];
```

Comment:
- [ ] **Required:** Great job implementing some suggested changes by the previous reviewers. However, your app does POST because you are sending the wrong format to the API endpoint.

Kindly change the below

https://github.com/Aduda-Boaz/bookstore/blob/849d451d831b6f249acc7037a1b56dcb1bbec1d4/src/components/BookForm.js#L17-L24

to

```
const newBook = {
        item_id: uuidv4(),
        title,
        author,
        category,
      };
```

Reply:
Thank you, made the changes and it works well.